packages/squiggle-lang/src/PointSet/MixedPoint.ts

```diff
@@ -1,19 +1,26 @@
+// MixedPoints are used to represent PDFs and scores.
 export type MixedPoint = {
   continuous: number;
   discrete: number;
 };
 
-export const makeContinuous = (f: number): MixedPoint => ({
-  continuous: f,
-  discrete: 0,
-});
+export function makeContinuous(f: number): MixedPoint {
+  return {
+    continuous: f,
+    discrete: 0,
+  };
+}
```

Comment:
why the change?

Reply:
It's just a coding style change, I usually don't do this unless I want to do other changes, but here I _thought_ I'll make them and then forgot about it.

From https://github.com/quantified-uncertainty/squiggle/blob/main/CONTRIBUTING.md:

> Prefer `function` keyword over arrow functions in top scope, use arrow functions otherwise

(trivial to do in VS Code, btw, `Cmd+.` -> "Convert to named function".)